.github/workflows/ci.yml

```diff
@@ -85,27 +86,35 @@ jobs:
       - name: Checkout
         uses: actions/checkout@master
 
+      - name: "Install PHP with extensions"
+        uses: shivammathur/setup-php@master
+        with:
+          php-version: ${{ matrix.php-version }}
+          coverage: xdebug
+          extension-csv: dom, json, libxml, mbstring, soap, xml, xmlwriter
```

Comment:
Since `shivammathur/setup-php` allows it, I have also enabled the `soap` extension.

Would you prefer it to be enabled in a separate pull request?